docs/Alerts.md

```diff
@@ -49,13 +49,13 @@ The following describes the proposed alerts for our ngsa pre-prod environment ba
 
   # Failed Client Requests (ngsa-cosmos)
 
-  fbwebv_CL
```

Comment:
@aflinchb - please take a look at these changes.  They won't work now, but, I think after we deploy the new cluster, the table name will be correct

Reply:
We will have to update these queries anyway to match the new logs (both in docs and in the actual alerts).  I don't think "AppPodType" exists anymore (will need to infer from the server name).